samples/drivers/led_max7219/src/main.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2015 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <gpio.h>
+#include <spi.h>
+
+#define SPI_DEV  DT_SPI_1_NAME
+#define GPIO_DEV DT_GPIO_P0_DEV_NAME
```

Comment:
Can you make main.c non-board-specific, please?

Reply:
I believe it's fixed, please take a look now.